portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -30,10 +33,16 @@
 @WebServlet("/data")
 public class DataServlet extends HttpServlet {
   
-  private List<String> messages = new ArrayList<String>();
+  private Query query = new Query("Comment");
```

Comment:
I meant to have DataStoreService be the instance variable.

Please initialize that in init(), not during construction of DataServlet. This query can be inlined, as I suggested